Item(by='lordgilman', descendants=None, kids=None, score=None, time=1601939713, title=None, item_type='comment', url=None, parent=24692729, text='The issue was noticed before 3.9 came out, but I guess it wasn&#x27;t a big enough deal to blog the release. Hopefully it&#x27;s just a measurement problem.<p><a href="https:&#x2F;&#x2F;bugs.python.org&#x2F;issue39117" rel="nofollow">https:&#x2F;&#x2F;bugs.python.org&#x2F;issue39117</a>')